momentum_scanner_intraday.py

```diff
@@ -1,11 +1,17 @@
 import time
-
+import json
 import pandas as pd
-
+import dex_chain
 import cache_db
 import gecko
 import metrics
 
+# NETWORK_MAP_CG: map a network to their CG id
+NETWORK_MAP_CG = {'Ethereum':'ethereum','BSC':'binance-smart-chain','Polygon':'polygon-pos','Arbitrum':'arbitrum-one'}
+# NETWORK_MAP_CGTERMINAL: map a network to their CG id
+NETWORK_MAP_CGTERMINAL = {'Ethereum':'eth','BSC':'bsc','Polygon':'polygon_pos','Arbitrum':'arbitrum'}
+DEX_CHAIN = dex_chain.DEX_CHAIN
```

Comment:
Either move these constants into `dex_chain.py` or inline them here. You can just import them `DEX_CHAIN = dex_chain.DEX_CHAIN` is not necessary.

Reply:
Moved into dex_chain.py